packages/components/src/components/domainproperties/samples/SampleTypeDesigner.tsx

```diff
@@ -458,8 +343,10 @@ export class SampleTypeDesigner extends React.PureComponent<Props, State> {
         }
 
         saveDomain(domainDesign, Domain.KINDS.SAMPLE_TYPE, details, name)
-            .then((savedDomain) => {
-                this.onFinishSuccess(savedDomain);
+            .then((response: DomainDesign) => {
```

Comment:
Doesn't look like we are logging errors that come from saveDomain failing.  Can we do that in saveDomain so we don't need to sprinkle it throughout all of these calls?

Reply:
Yes. Done.